pac4j-oidc/src/main/java/org/pac4j/oidc/config/OidcConfiguration.java

```diff
@@ -178,6 +178,10 @@ public void setClientAuthenticationMethod(final ClientAuthenticationMethod clien
         this.clientAuthenticationMethod = clientAuthenticationMethod;
     }
 
+    public void setClientAuthenticationMethodAsString(String auth) {
+        this.clientAuthenticationMethod.parse(auth);
```

Comment:
I don't think this will work because the returned and parsed value is not used. I expected: `this.clientAuthenticationMethod = ClientAuthenticationMethod.parse(auth);`